Core/Commands/CustomCommands.py

```diff
@@ -619,3 +619,21 @@ def subreddit(bot, event, *args):
                                                              hangups.SegmentType.LINK,
                                                              link_target=link_url)])
 
+@DispatcherSingleton.register
+def subverse(bot, event, *args):
```

Comment:
Perhaps you could also expand upon this command by adding in a autoreply that triggers from /v/*? So that way users could go /v/subverse and it would also link?